tests/actor/power_distributing/test_distribution_algorithm.py

```diff
@@ -218,6 +237,31 @@ def test_distribute_power_two_batteries_2(self) -> None:
         assert result.distribution == approx({1: 200, 3: 400})
         assert result.remaining_power == approx(0.0)
 
+    def test_distribute_power_two_batteries_one_inverter(self) -> None:
+        """Test when the batteries has different SoC and only one inverter.
+
+        No matter the differences, the power should be distributed equally,
+        because there is only one inverter.
+        """
+        capacity: list[float] = [49000, 98000]
+        components = self.create_components_with_capacity(
+            2, capacity, share_inverter=True
+        )
+
+        available_soc: dict[int, float] = {0: 20, 2: 20}
+        incl_bounds: dict[int, float] = {1: 500}
+        excl_bounds: dict[int, float] = {
+            1: 0,
+        }
+
+        algorithm = DistributionAlgorithm(distributor_exponent=1)
+        result = algorithm._distribute_power(  # pylint: disable=protected-access
+            components, 600, available_soc, incl_bounds, excl_bounds
+        )
+
+        assert result.distribution == approx({1: 500})
+        assert result.remaining_power == approx(100.0)
+
```

Comment:
I guess you need some tests for the distribution algorithm with exclusion bounds as well,  there's a class for those tests at the end of this file I think.